src/components/CompletedRow.jsx

```diff
@@ -1,17 +1,31 @@
-import { Grid, Segment } from 'semantic-ui-react';
+import { Grid, Icon, Label, Segment } from 'semantic-ui-react';
 import TrainBullet from './TrainBullet';
 import { checkGuessStatuses } from '../utils/answerValidations';
+import { loadSettings } from '../utils/settings';
 
 const CompletedRow = (props) => {
   const { guess } = props;
   const classNameArrays = checkGuessStatuses(guess)
+  const settings = loadSettings();
+
   return (
     <Grid.Row>
       {
         guess.map((routeId, index) => {
           return (
             <Grid.Column key={`guess-${index}`}>
               <Segment placeholder className={classNameArrays[index]}>
+                {settings.display.showAnswerStatusBadges &&
+                  <Label as='a' floating circular size='tiny'>
+                    {
+                      classNameArrays[index] === 'present' ?
+                        <Icon name="arrows alternate horizontal" fitted /> :
+                        classNameArrays[index] === 'correct' ?
+                          <Icon name="check" fitted /> :
+                          <Icon name="close" fitted />
```

Comment:
```suggestion
                          <Icon name="x" fitted />
```

Looks like this would make the "x" icon black instead of gray.